### ISSUE TITLE
Add filtering by project

### ISSUE THREAD

**Issue by peckjj** [peckjj/RemindMe#3]
Add a `--project` option to allow filtering based on a specified project.

Add the full logic for querying tasks by description and filtering by project when this command is present

**Comment by sourvad** [comment]
So is this enhancement to query the task description to find if the specified keyphrase is present?

**Comment by peckjj** [comment]
@sourvad 
Hey, you checked out the repo!

No, querying tasks by description for similar text is already implemented. This is the primary function of the `get` command. For example, using the following CLI command will query and fetch all tasks with the letters "abc" anywhere in the description:

`reme get abc`

This is implemented by using a `LIKE` SQL comparison operator.

This enhancement is just like filtering on "tags", although I called them "projects" instead. Projects, like tags, allow the user to categorize or group tasks together that are related in some way so that they can query for them easier.

I realize that I didn't really put much effort into writing the requirements for this issue, so I'll elucidate now. The desired behavior is to implement a new CLI option `--project` (with aliases like `--proj` or `-p`) that will accept a `string` project name as its parameter. An example command after it is implemented is thus:

`reme get abc --project work`

The above command should query all tasks that have 'abc' anywhere in its description AND belong to the 'work' project.

Here's a list of bulleted reqs:

- New `--project` option added (with aliases) that takes a project name as its parameter
- When using the `reme add` command with the `--project` option, the task that is added to the DB will be assigned the project that was given
- When using the `reme get` command with the `--project` option, only the tasks assigned to that project will be fetched.
- Project names will be case sensitive and must match exactly. No need to use a `LIKE` SQL comparison, just use direct equality comparison, like so: `SELECT * FROM tasks WHERE project = {project}`.


If you decide to take up this issue, hopefully you find my arg parsing system easy enough to use and build with. I'm happy to answer any questions if you have them. You'll also find that my design choices are not optimal, currently. I looked over the `Database.cs` class and found that the `GetTaskByPrio()`  function might have a pattern that is easy to add new filters onto. In fact, the `GetTaskByDesc()` function could probably be removed, so that only one function handles filtering on all task properties. I'd consider that approach first.
